### PR TITLE
Fix carousel width issue in firefox

### PR DESCRIPTION
**Pull request by cristain7**

### CODE REVIEW

File: src/components/image_carousel/image_carousel.vue [hunk 1]
```diff
@@ -1,32 +1,36 @@
...
-        <div class="osg-carousel__slides" ref="slides">
-          <figure v-for="(image, index) in images" v-bind:key="index">
-            <picture>
-              <source :srcset="image.large" media="(min-width: 1024px)" />
-              <source :srcset="image.medium" media="(min-width: 769px)" />
-              <source :srcset="image.small" media="(max-width: 768px)" />
-              <img :src="image.small" :alt="image.caption" />
-            </picture>
-          </figure>
+    <div class="osg-grid">
```
Comment (larsboldt) [inline src/components/image_carousel/image_carousel.vue:3]: It is not allowed to use grid anymore in components. If a grid is needed you must create a new one for that specific component. Grid can only be used on custom content.

**Review by kbergha — APPROVED** [review]

File: src/components/image_carousel/image_carousel.scss [hunk 2]
```diff
@@ -8,6 +8,17 @@ $osg-carousel-image-height-large: 725;
 .osg-carousel {
   position: relative;
 
+  &__wrapper {
```
Comment (larsboldt) [inline src/components/image_carousel/image_carousel.scss:11]: Do we need the extra divs for this? If you move css grid code from wrapper to osg-carousel and set the container code for grid-column on both content and carousel__info - will you not achieve the same result with less markup?

Comment (cristain7) [reply src/components/image_carousel/image_carousel.scss:13]: That is correct! I changed that :)

**Review by jeanettelandoyaune — APPROVED** [review]

File: src/components/image_carousel/image_carousel.scss [hunk 3]
```diff
@@ -6,9 +6,13 @@ $osg-carousel-image-height-small: 290;
 $osg-carousel-image-height-medium: 540;
 $osg-carousel-image-height-large: 725;
 .osg-carousel {
+  display: grid;
+  grid-template-columns: repeat(12, 1fr);
   position: relative;
 
   &__content {
+    grid-column-end: span 12;
```
Comment (larsboldt) [inline src/components/image_carousel/image_carousel.scss:14]: Start and end can be simplified to grid-column: span 12 / span 12;

But should you also not add osg-carousel__info to this?

Comment (cristain7) [reply src/components/image_carousel/image_carousel.scss]: Stemmer!

**Review by larsboldt — APPROVED** [review]

**Pull request merged**